### ISSUE TITLE
Freebsd make sample project fails due to missing GLIBC_2.14

### ISSUE THREAD

**Issue by devingDev** [vitasdk/autobuilds#3]
I followed the setup as in vitasdk.org and set everything accordingly. After that I cloned the samples git and tried running "make" inside hello_world.
It says that GLIBC_2.14 was not found and stops.

**Comment by endrift** [comment]
I don't believe that the autobuilds support FreeBSD. You have to build it manually using the buildscripts. I unofficially maintain FreeBSD support for those so let me know if you have any trouble building it.
